src/runtime/go-test.sh

```diff
@@ -167,22 +156,13 @@ main()
 		shift
 	done
 
-	run_coverage=no
-	if [ "$CI" = true ] || [ -n "$KATA_DEV_MODE" ]; then
-		run_coverage=yes
-	fi
-
 	local go_ldflags
 	[ "$(go env GOARCH)" = s390x ] && go_ldflags="-extldflags -Wl,--s390-pgste"
 
 	# KATA_GO_TEST_FLAGS can be set to change the flags passed to "go test".
 	go_test_flags=${KATA_GO_TEST_FLAGS:-"-v $race -timeout $timeout_value -ldflags '$go_ldflags'"}
 
-	if [ "$run_coverage" = yes ]; then
-		test_coverage
-	else
-		test_local
-	fi
+	test_coverage
```

Comment:
I'm wondering if there is any (bad) implications of running coverage tests on a release build. If any, that can explain the existence of the `test_local()`.

Reply:
> I'm wondering if there is any (bad) implications of running coverage tests on a release build. If any, that can explain the existence of the `test_local()`.

I don't see how.  This is *only* changing options to `go test` - we're not changing anything about the build itself here.